internal/lib/lib.go

```diff
@@ -1166,7 +1166,6 @@ func InformersToRegister(kclient *kubernetes.Clientset, oclient *oshiftclient.Cl
 		utils.EndpointInformer,
 		utils.SecretInformer,
 		utils.ConfigMapInformer,
-		utils.PodInformer,
 	}
 
 	if GetServiceType() == NodePortLocal {
```

Comment:
@akshayhavile code was present to enable podInformers only when the `serviceType` is `NodePortLocal`. But in L1169, it was added directly without any check which I removed.